lib/matplotlib/backend_bases.pyi

```diff
@@ -76,7 +76,7 @@ class RendererBase:
         facecolors: Sequence[ColorType],
         antialiased: bool,
         edgecolors: Sequence[ColorType] | ColorType | None,
-    ): ...
+    ) -> None: ...
```

Comment:
Slightly odd that the implementation has a `return ` statement, but I do agree that `draw_path_collection` also returns `None`, so I guess we _should_ propegate that. Also just consistent with other `draw_` methods